content/zh-cn/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md

```diff
@@ -51,7 +51,7 @@ Kubernetes cluster.
     worker2   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker2
     ```
     <!--
-    1. Chose one of your nodes, and add a label to it:
+    1. Choose one of your nodes, and add a label to it:
```

Comment:
Seems the ordered list is not correct now:

<img width="757" alt="image" src="https://user-images.githubusercontent.com/79828097/225791997-0d97aab0-049b-4513-8232-c81b131b94f8.png">

Reply:
it's wired...